server/routerlicious/packages/routerlicious-base/src/utils/sessionHelper.ts

```diff
@@ -59,10 +59,12 @@ async function updateExistingSession(
     deltaStreamUrl: string,
     document: IDocument,
     existingSession: ISession,
-    documentId,
+    documentId: string,
+    tenantId: string,
     documentsCollection: ICollection<IDocument>,
     sessionStickinessDurationMs: number,
     lumberjackProperties: Record<string, any>,
+    count: number,
```

Comment:
A nit: we can probably make this have the default value of 0 so that it doesn't have to be passed in documents.ts

Reply:
Got it. Done. :)

Reply:
It should be set at line 217.